packages/sdk/src/contracts/prebuilt-implementations/edition-drop.ts

```diff
@@ -225,6 +228,19 @@ export class EditionDrop extends StandardErc1155<DropERC1155> {
     return this.erc1155.totalCount();
   }
 
+  /**
+   * Get current owner of the contract
+   *
+   * @example
+   * ```javascript
+   * const owner = await contract.getOwner();
+   * ```
+   * @returns The owner address.
+   */
+  public async getOwner(): Promise<string> {
+    return this.owner.get();
```

Comment:
the pattern we've been doing is directly accessing these common functions using the namespace.

`owner` is public so ppl can call `contract.owner.get()` and `contract.owner.set()`

no need to re-expose these at the top level

Reply:
I deployed all the pre-built contracts with the dashboard to check in the explorer section to make sure it contains `setOwner` and `owner`.

- Edition
- Edition Drop
- NO - Marketplace
- Multiwrap
- NFT Collection
- NFT Drop
- Pack
- Signature Drop
- NO - Split
- NO - Token
- NO - Token Drop
- NO - Vote

Few contracts doesn't have owner so I implemented only on those that have.
I re-exposed at the top level as requested in the goals in the issue #81